src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -225,8 +225,8 @@ namespace AirLoopHVACDOAS {
                 ++AirLoopMixerNum;
                 AirLoopMixer thisMixer;
 
-                thisMixer.name = UtilityRoutines::MakeUPPERCase(thisObjectName);
-                thisMixer.OutletNodeName = UtilityRoutines::MakeUPPERCase(fields.at("outlet_node_name").get<std::string>());
+                thisMixer.name = UtilityRoutines::makeUPPER(thisObjectName);
```

Comment:
Just did a global `sed` to shorten this name. You're welcome. I have another fun name shortener here also.